src/components/button/button_group/_button_group_button.scss

```diff
@@ -123,20 +123,43 @@
 .ouiButtonGroup--small {
   .ouiButtonGroupButton {
     border: $ouiBorderThin;
+  }
```

Comment:
Why dont we want border thin for vertical?

Reply:
both vertical and horizontal has `$ouiBorderThin` which set on line 125.

Reply:
Oops yeah my bad